src/SplitIO/Engine.php

```diff
@@ -31,7 +32,21 @@ public static function getTreatment($matchingKey, $bucketingKey, SplitGrammar $s
             self::EVALUATION_RESULT_LABEL => null
         );
 
+        $inRollOut = false;
         foreach ($conditions as $condition) {
+            if (!$inRollOut  && $condition->getConditionType() == ConditionTypeEnum::ROLLOUT) {
+                if ($split->getTrafficAllocation() < 100) {
+                    $bucket = Splitter::getBucket(
+                        $split->getAlgo(),
+                        $split->getKey(),
+                        $split->getTrafficAllocationSeed()
+                    );
+                    if ($bucket >= $split->getTrafficAllocation()) {
+                        return $split->getDefaultTratment();
```

Comment:
@mredolatti This method is returning the Array named $result with 2 keys, one of this is for the treatment result and the other one is for the label. So, I think that return value should be consistent.